test-requirements.txt

```diff
@@ -1,3 +1,4 @@
+importlib-metadata==4.13.0
```

Comment:
Could you add a comment pointing to this PR and saying that it could be removed in the future?

Reply:
sure